### ISSUE TITLE
Pymf Cannot be Found When Freezing with PyInstaller

### ISSUE THREAD

**Issue by slalomchip** [pvys/CV-camera-finder#4]
CV-camera-finder works great from within an IDE such as Spyder.  But a baseline frozen with PyInstaller throws the error:

`ImportError: DLL load failed while importing pymf: The specified module could not be found.`

`pymf.pyd` is in the same folder at the frozen executable file, `python3.dll`, and `python38.dll`, so I think `pymf.pyd` is located where it should be.

Are you aware of issues or solutions when including `pymf.pyd` in a baseline frozen with PyInstaller?

**Comment by pvys** [comment]
Hi @slalomchip!
Sadly, when I was making this function, I was not aware of these issues. Anyway, as this function depends on the VS redistributable, I think including the path of it may help.

**Comment by slalomchip** [comment]
@pvys  - Thanks for the response!
I was trying to create the frozen baseline using Python 3.8.  After learning how to debug imports in PyInstaller, I learned that pymf.pyd wants to import python37.dll (which it obviously can't find in Python 3.8).  I downgraded to Python 3.7 and my issue is now resolved.  Is there a way that msmf.pyd could be modified to make it independent of which Python version is required?

Thanks for creating msmf.pyd!  It solves a problem I couldn't find a solution for.
